modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx

```diff
@@ -558,6 +562,24 @@ function AggregationSourceProperty({
 		ObjectField[]
 	>();
 
+	const filteredObjectRelationships = useMemo(() => {
+		return objectRelationships?.filter(({label}) => {
+			return stringIncludesQuery(
```

Comment:
@danvb112 You can remove the `{}` for the second return. It can make the code more lean.